transports/transport-rest/transport-rest-vertx/src/main/java/io/servicecomb/transport/rest/vertx/accesslog/element/impl/BytesWrittenV2Element.java

```diff
@@ -0,0 +1,25 @@
+package io.servicecomb.transport.rest.vertx.accesslog.element.impl;
+
+import io.servicecomb.transport.rest.vertx.accesslog.AccessLogParam;
+import io.servicecomb.transport.rest.vertx.accesslog.element.AccessLogElement;
+import io.vertx.core.http.HttpServerResponse;
+
+public class BytesWrittenV2Element implements AccessLogElement {
```

Comment:
why do we have two versions of bytes written element?

Reply:
They will return different result when bytes written is zero. v1 is "0" and v2 is "-".